numpy/_core/meson.build

```diff
@@ -953,6 +953,7 @@ foreach gen_mtargets : [
     [
       AVX512_SKX, [AVX2, FMA3],
       NEON_VFPV4,
+      VXE2, VXE
```

Comment:
need a re-base maybe? ppc64 features `VSX4, VSX3, VSX2` are missing???

Reply:
Yeah looks like I made a mess of the rebase :/. It should be fixed now.